dash_app/dashboard.py

```diff
@@ -108,28 +105,33 @@ def init(server):
         children=[
             html.H1(children="CAN Data Pipeline Dashboard"),
             html.P(commit_str),
-            html.H2("Time-series count"),
-            html.H3("By variable group"),
-            dash_table_from_data_frame(
-                agg_level_and_field_group.has_timeseries, id="agg_has_timeseries"
+            html.H2("Time-series pivot table"),
```

Comment:
nit: Hyphen looks weird.  I think "Time series" is more common than "Time-series".